AGOLAccountRequestor/agol_auth.py

```diff
@@ -5,6 +5,27 @@
 from accounts.models import AGOL, AGOLUserFields
 
 
+def get_user_details(response):
+    if 'error' in response:
+        return {}
+
+    first_name = response.get('firstName', '')
+    last_name = response.get('lastName', '')
+    full_name = response.get('fullName', '')
+    if full_name != '' and first_name == '' and last_name == '':
+        full_name = full_name.split(' ')
+        first_name = full_name[0]
+        last_name = ' '.join(full_name[1:])
+
+    return {
+        'username': response.get('username'),
+        'email': response.get('email'),
```

Comment:
Is there any value in continuing to return full_name as part of this object, or is it completely pointless if we consistently return first and last names?  
Also... for those cases where we see lastname, firstname, is that just full_name, and so should we also check for a comma as the last character of full_name[0], and if so, we drop that character and swap the order?

Reply:
Full name is a calculated value on the internals of django based on first and last names so I'm not clear that it was ever doing anything. 
Yes if the comma format is